src/index.js

```diff
@@ -1,13 +1,13 @@
 let _ = require('lodash/fp')
-let Promise = require('bluebird')
```

Comment:
I think this means we can drop bluebird from the whole lib now. `tap` was the only nonstandard bluebird API we were using, and that's gone now

Reply:
yup, done